tests/test_minimal/test_tools/test_backend_and_dataset_configuration/test_helpers/test_configure_backend_overrides.py

```diff
@@ -74,3 +72,44 @@ def test_simple_time_series_override(
             assert written_data.compression_opts == higher_gzip_level
         elif backend == "zarr":
             assert written_data.compressor == numcodecs.GZip(level=5)
+
+
+@pytest.mark.parametrize("backend", ["hdf5", "zarr"])
+def test_simple_dynamic_table_override(tmpdir: Path, backend: Literal["hdf5", "zarr"]):
+    data = np.zeros(shape=(30_000 * 5, 384), dtype="int16")
+
+    nwbfile = mock_NWBFile()
+    dynamic_table = DynamicTable(
+        name="TestDynamicTable", description="", columns=[VectorData(name="TestColumn", description="", data=data)]
+    )
+    nwbfile.add_acquisition(dynamic_table)
+
+    backend_configuration = get_default_backend_configuration(nwbfile=nwbfile, backend=backend)
+    dataset_configuration = backend_configuration.dataset_configurations["acquisition/TestDynamicTable/TestColumn/data"]
+
+    smaller_chunk_shape = (30_000, 64)
+    dataset_configuration.chunk_shape = smaller_chunk_shape
+
+    higher_gzip_level = 5
+    if backend == "hdf5":
```

Comment:
This if is unecessary.

Reply:
Oh, I remember you had this in another PR. I think you like it like this? I guess the code feels more extensible / general this way even if it not used. Well, you won't find me arguing for reducing lines as a virtue so I guess this is fine.

Reply:
Yes, this 4 line block is kind of silly but important to note how specific it is to GZIP syntax; if/when more tests are added for other compressors they will not look like this

Granted this is also something that was separated into different functions before the combination with parametrize, where each individual function would have been the extensible thing

So the if/else stuff here just because it's all together in one function

Reply:
Sure, you are keeping it as a template for how the tests should be. Fair enough.